astro/src/content/docs/get-started/download-and-install/air-gapping.mdx

```diff
@@ -0,0 +1,139 @@
+
+Please consult your friendly neighborhood network engineer for specific details about how to block requests to FusionAuth hostnames in your environment.
+
+## Adding Your License to Your Instance
+
+Purchase a license by [contacting the sales team](/contact). While other licenses are available on the [pricing page](/pricing), an air-gapped license requires a conversation with our sales team.
+
+Once a license is purchased, activate the instance on the license. 
+
+To do so, log into your [Account](https://account.fusionauth.io/account).  Navigate to the <Breadcrumb>Plan</Breadcrumb> tab.
+
+![View the plan tab.](/img/docs/get-started/download-and-install/plan-tab.png)
+
+Copy the appropriate license key and text. Use the "Production" license for your production server. The other license is suitable for non-production environments, such as user acceptance testing or development. 
+
+You can always find your license key and text by logging in to your [Account](https://account.fusionauth.io/account) and then navigating to the <Breadcrumb>Plan</Breadcrumb> tab.
+
+After you have your license key and text, log in to your FusionAuth instance. The credentials you use to log into the instance have no connection to the credentials you used to log into your Account Portal. You will need either the `admin` or `reactor_manager` roles in the FusionAuth Admin UI application to view the <Breadcrumb>Reactor</Breadcrumb> tab.
+
+<Aside type="caution">
+  Ensure the instance's network connection has been blocked, as documented in the [Preparing Your Instance](#preparing-your-instance) section. Otherwise, under certain circumstances, you may end up with a regular license replacing your air-gapped license.
+
+  Please see this [GitHub issue for more information](https://github.com/FusionAuth/fusionauth-issues/issues/3105).
+</Aside>
+
+Navigate to the <Breadcrumb>Reactor</Breadcrumb> tab and enter your license key and text in the <InlineField>License key</InlineField> field.
+
+Check the <InlineField>I have an air-gapped license</InlineField> box underneath the <InlineUIElement>License Key</InlineUIElement> field. Add the license text into the <InlineField>License text</InlineField> field.
+
+![Activate your license using Reactor.](/img/docs/get-started/download-and-install/air-gapped-license-activate.png)
+
+Immediately after activating, FusionAuth validates the license. It may take a minute or two to complete activation. No network access is required for activation.
+
+Once that has happened, the <InlineField>Licensed</InlineField> field will change to a green checkmark.  This may require a page refresh. You will also see a list of premium features and the expiration date of the license. Depending on your particular plan, some features may not be active or available to you. You can see a list of features on the [premium features page](/docs/get-started/core-concepts/premium-features).
```

Comment:
```suggestion
Once that has happened, the <InlineField>Licensed</InlineField> field will change to a green checkmark.  This may require a page refresh (F5). You will also see a list of premium features and the expiration date of the license. Depending on your particular plan, some features may not be active or available to you. You can see a list of features on the [premium features page](/docs/get-started/core-concepts/premium-features).
```